_opcodes/op293-bg2.html

```diff
@@ -0,0 +1,21 @@
+---
+n: 293
+name: "Script: Enable Offscreen AI"
+param1: "Irrelevant"
+param2: "Constant Value"
+bg1: 0
+bg2: 1
+bgee: 0
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+The targeted creature(s) will still run scripts when off-screen and out of combat.
+<br/>
+<br/>The effect modifies stat 183 to param2.
+
+{% capture note %}
+The Constant Value parameter should be set to 1.
+{% endcapture %}
+
+{% include important.html %}
```

Comment:
and all stat modifiers in this list, so I don't spam you with comments.

Reply:
> and all stat modifiers in this list, so I don't spam you with comments.

Done.

Please double check everything is fine...